core/editor.js

```diff
@@ -2228,3 +2305,53 @@ CKEDITOR.ELEMENT_MODE_INLINE = 3;
  * @event contentDomInvalidated
  * @param {CKEDITOR.editor} editor This editor instance.
  */
+
+/**
+ * Allow delayed editor creation (either with callback or intervals) if target element is detached from DOM.
+ *
+ *		config.delayIfDetached = true;
+ *
+ * @since 4.17.0
+ * @cfg {Boolean} [delayIfDetached=false]
+ * @member CKEDITOR.config
+ */
+CKEDITOR.config.delayIfDetached = false;
+
+/**
+ * Function with single argument. As argument is passed another function that continues editor creation.
+ * Allows to store the editor creation function and invoke it whenever it is convenient instead of periodically
+ * check if element is reattached to DOM.
```

Comment:
> Btw. what will happen if the function is called but the editor element is still detached? I see from the code it will simply try to create editor instance. This is good (I mean the 3rd-party controls it), however it will be good to mention here too - that the function should be called only after editor element is reattached to DOM.

Generally, it will be the same situation as we have now. 

I will improve doc comments.